lib/spack/spack/modules/common.py

```diff
@@ -133,9 +133,9 @@ def dependencies(spec, request='all'):
         the direct dependencies if request is 'direct', or the entire DAG
         if request is 'all'.
     """
-    if request not in ('none', 'direct', 'all'):
+    if request not in ('none', 'direct', 'external', 'all'):
```

Comment:
to avoid needing to keep things in sync:
`allowed_requests = ('none', 'direct', 'external', 'all')`
and reuse below?

Reply:
keeping the diff to upstream as small as possible, if they have the list twice, I kept it that way.

Reply:
that was a theory I had, coolio